.github/scripts/hub-util.sh

```diff
@@ -0,0 +1,1045 @@
+#!/bin/bash
```

Comment:
Why is this script being copied into this repo (from https://github.com/kata-containers/.github/blob/main/scripts/hub-util.sh)?

If we only want it here (and I'm not sure we do as we have other repos), that needs to be documented in the commit, and an issue raised in the `.github` repo to delete the original version of this script.

Reply:
I copied it to this repo to make it possible to test it before it was merged into that repo, since the action pulls it from github. I planned on deleting it once that script was merged in.